doc/audio.md

```diff
@@ -1,8 +1,17 @@
 # Audio
 
-To play an audio is really simple!
+To play audio is really simple!
 
-Start by adding an entry in your `pubspec.yaml` for each file you want to use (and make sure that the files exist in the paths that you provide).
+First you have to add [flame_audio](https://github.com/flame-engine/flame_audio) to your dependency
+list in your `pubspec.yaml` file:
+
+```yaml
+dependencies:
+  flame_audio: 0.1.0-rc4
```

Comment:
Same here as I mentioned in `doc/README.md`, we will forget to update this.. Either reference to the install instructions, or just say "add flame_audio to your pubspec.yaml".